### ISSUE TITLE
Testing needs to be done asynchronously

### ISSUE THREAD

**Issue by c3ypt1c** [c3ypt1c/PasswordManager#4]
Current implementation could benefit from asynchronicity.

**Comment by c3ypt1c** [comment]
This depends on #10. If tests aren't done, it doesn't matter how you do them.

**Comment by c3ypt1c** [comment]
Testing is done asynchronously but the page still blocks?

**Comment by c3ypt1c** [comment]
Re-implement with [WebWorkers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers).

**Comment by c3ypt1c** [comment]
Added temporary won't fix to this issue as it doesn't stop the project from working
